File: 151002/Kragel/distributed/publisher/src/main/java/by/bsuir/publisher/model/Sticker.java
package by.bsuir.publisher.model;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

import java.util.ArrayList;
import java.util.List;

@Getter
@Setter
@Entity
@Table(name = "tbl_sticker")
public class Sticker {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    private Long id;

    @Column(name = "name", nullable = false, unique = true, length = 32)
    private String name;


    @ManyToMany(cascade = CascadeType.ALL)
    @JoinTable(name = "tbl_sticker_tweets",
            joinColumns = @JoinColumn(name = "sticker_id"),
            inverseJoinColumns = @JoinColumn(name = "tweets_id"))
    private List<Tweet> tweets = new ArrayList<>();

}

File: 151003/Klimovich/Publisher/Models/Sticker.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace Publisher.Models
{
    [Table("tbl_sticker")]
    public class Sticker
    {
        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Required]
        public int id { get; set; }

        [MaxLength(32)]
        [Required]
        public string name { get; set; }

        public List<Tweet> Tweets { get; set; } = new();
        public List<TweetSticker> TweetStickers { get; set; } = new();

    }
}
